frontend/src/modules/WellLogViewer/view/components/SubsurfaceLogViewerWrapper.tsx

```diff
@@ -169,18 +175,29 @@ export function SubsurfaceLogViewerWrapper(props: SubsurfaceLogViewerWrapperProp
     });
     const colorTables = React.useMemo(() => createContinuousColorScaleForMap(colorScale), [colorScale]);
 
-    // Global value syncronization
-    const broadcastGlobalMdChange = useGloballySyncedMd(
-        props.wellboreHeader,
-        wellLogController,
+    const instanceId = props.moduleProps.viewContext.getInstanceIdString();
+    const syncableSettingKeys = props.moduleProps.viewContext.useSyncedSettingKeys();
+    const wellboreUuid = props.wellboreHeader?.wellboreUuid ?? "";
+
+    // Set up global hover md synchronization
+    useSubscribeToGlobalHoverMdChange(props.moduleProps.workbenchServices, wellLogController, instanceId, wellboreUuid);
```

Comment:
I think this is much more comprehensive now :smiley:

Reply:
👍